testing_and_setup/compass/ocean/cosine_bell/RMSE/default/compute_rmse.py

```diff
@@ -1,76 +1,85 @@
             latCent = float(line[ii:])
-        if "config_cosine_bell_lon_center" in line: 
+        if "config_cosine_bell_lon_center" in line:
             ii = line.find('=')+1
             lonCent = float(line[ii:])
-        if "config_cosine_bell_radius" in line: 
+        if "config_cosine_bell_radius" in line:
             ii = line.find('=')+1
             radius = float(line[ii:])
-        if "config_cosine_bell_psi0" in line: 
+        if "config_cosine_bell_psi0" in line:
             ii = line.find('=')+1
             psi0 = float(line[ii:])
-        if "config_cosine_bell_vel_pd" in line: 
+        if "config_cosine_bell_vel_pd" in line:
             ii = line.find('=')+1
             pd = float(line[ii:])
 
-    init = xr.open_dataset('../../../{}/default/init_step/initial_state.nc'.format(resTag))
-    #find time since the beginning of run
-    ds = xr.open_dataset('../../../{}/default/forward/output/output.0001-01-01_00.00.00.nc'.format(resTag))
+    init = xr.open_dataset('../../../{}/default/init_step/initial_state.nc'
+                           ''.format(resTag))
+    # find time since the beginning of run
+    ds = xr.open_dataset('../../../{}/default/forward/output/'
+                         'output.0001-01-01_00.00.00.nc'.format(resTag))
     for j in range(len(ds.xtime)):
-      tt=str(ds.xtime[j].values)
-      tt.rfind('_')
-      DY = float(tt[10:12])-1
-      if DY == pd:
-        sliceTime = j
-        break
+        tt = str(ds.xtime[j].values)
+        tt.rfind('_')
+        DY = float(tt[10:12])-1
+        if DY == pd:
+            sliceTime = j
+            break
     HR = float(tt[13:15])
     MN = float(tt[16:18])
     t = 86400.0*DY+HR*3600.+MN
-    #find new location of blob center
-    #center is based on equatorial velocity
+    # find new location of blob center
+    # center is based on equatorial velocity
     R = init.sphere_radius
     distTrav = 2.0*3.14159265*R / (86400.0*pd)*t
-    #distance in radians is
+    # distance in radians is
     distRad = distTrav / R
     newLon = lonCent + distRad
     if newLon > 2.0*np.pi:
         newLon -= 2.0*np.pi
 
-    #construct analytic tracer
-    tracer = np.zeros_like(init.tracer1[0,:,0].values)
+    # construct analytic tracer
+    tracer = np.zeros_like(init.tracer1[0, :, 0].values)
     latC = init.latCell.values
     lonC = init.lonCell.values
-    temp = R*np.arccos(np.sin(latCent)*np.sin(latC) + np.cos(latCent)*np.cos(latC)*np.cos(lonC - newLon))
+    temp = R*np.arccos(np.sin(latCent)*np.sin(latC) +
+                       np.cos(latCent)*np.cos(latC)*np.cos(lonC - newLon))
     mask = temp < radius
     tracer[mask] = psi0 / 2.0 * (1.0 + np.cos(3.1415926*temp[mask]/radius))
 
-    #oad forward mode data
-    tracerF = ds.tracer1[sliceTime,:,0].values
-    rmse = np.sqrt(np.mean((tracerF-tracer)**2))
+    # oad forward mode data
+    tracerF = ds.tracer1[sliceTime, :, 0].values
+    rmseValue = np.sqrt(np.mean((tracerF-tracer)**2))
```

Comment:
It is a little risky to use the name of a function as a variable name within that function in Python.  It is considered a re-definition since you could otherwise call the function recursively.  I think it's better to use a different name in this case.